lib/Reader/readerir.cpp

```diff
@@ -5630,11 +5626,11 @@ IRNode *GenIR::genCall(ReaderCallTargetData *CallTargetInfo, bool MayThrow,
     throw NotYetImplementedException("Direct unmanaged call");
   }
 
-  CallArgType ResultType = Signature.getResultType();
-  if (ResultType.CorType == CORINFO_TYPE_REFANY) {
-    throw NotYetImplementedException("Return refany");
+  if (this->MethodSignature.isVarArg()) {
+    throw NotYetImplementedException("Vararg method");
```

Comment:
You should put this check up around line 273 (didn't realize there were 2 places we looked for return refany).

Basically. we want to reject requests to jit varargs methods as soon as we know we have one, rather than spending time jitting parts of it only to bail out once we see it making a call. Aside from being a bit more efficient (not that that matters much if we're bailing out) there's the off chance that a varargs method doesn't make any calls at all.